pages/linux/gpasswd.md

```diff
@@ -0,0 +1,23 @@
+# gpasswd
+
+> Administer /etc/group and /etc/gshadow. Administer linux group.
```

Comment:
```suggestion
> Administer `/etc/group` and `/etc/gshadow`.
```

Perhaps this would read better? @agnivade, is this correct syntax?

Reply:
I think from previous PRs the correct syntax would be using quotes :thinking: Thought that backticks were for commands. But I may be wrong. :+1:

Reply:
Updated suggestion. Thanks, @pxgamer :smiley_cat: